analytics/lib/counts.py

```diff
@@ -1,5 +1,6 @@
 from django.db import connection, models
```

Comment:
Suggested commit message:
analytics: Add performance and transaction logging to counts.py.
For each database query made by an analytics function, log time spent and number of rows added. 
In the spirit of write ahead logging, for each (stat, end_time) update, log the start and end of the "transaction", and the total time spent.